debug/debugging.rst

```diff
@@ -56,5 +56,57 @@ by removing the call to ``loadClassCache()``::
     session.
 
 Some IDEs do not like the fact that some classes are stored in different
-locations. To avoid problems, you can tell your IDE to ignore the PHP cache
-file.
+locations. To avoid problems, you can either tell your IDE to ignore the PHP
+cache files, or you can change the extension used by Symfony for these files::
+
+    $kernel->loadClassCache('classes', '.php.cache');
+
+Useful debugging commands
+-------------------------
+
+When developing a large application, it can be hard to keep track of all the
+different services, routes and translations. Luckily, Symfony has some commands
+that can help you visualize and find the information.
+
+Information about the contents of the Symfony container can be found using the
+command:
+
+.. code-block:: terminal
```

Comment:
Lately we're trying to make docs more compact. Maybe we can merge the code-block + tip + code-block in just one code-block with comments? Something like this:

```
.. code-block:: terminal

    # displays all public services
    $ php bin/console debug:container

    # displays only the public services that contain
    # 'logger' as part of their names
    $ php bin/console debug:container logger
```